misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/mock/FunctionMock.java

```diff
@@ -102,49 +102,52 @@ private static Object callMockFunction(String originalFunction, String originalC
         String packageName;
         String version;
 
+        String[] projectInfo = Thread.currentThread().getStackTrace()[4].getClassName().split(Pattern.quote("."));
```

Comment:
Shall we add a check to see whether stacktrace element array size is >=5 before accessing the index?

Reply:
Added check for stacktrace